subham-rai/docker-go-simple-application/main.go

```diff
@@ -0,0 +1,10 @@
+package main
+
+ import "fmt"
+ import "myownfunctions"  // <-- remember to include the directory/package name - in this case "Project"
+
+ func main(){
+     vara := myownfunctions.SayHello() //<---- function name must be capital!
+     varb := myownfunctions.SayWorld()
+     fmt.Println(vara+" "+varb)
```

Comment:
why not use Printf for formatting?

Reply:
thanks for the suggestion...I'll keep this in mind.